test/ssl-tests/20-cert-select.cnf

```diff
@@ -215,7 +215,7 @@ PrivateKey = ${ENV::TEST_CERTS_DIR}/serverkey.pem
 [4-P-256 CipherString and Signature Algorithm Selection-client]
 CipherString = aECDSA
 MaxProtocol = TLSv1.2
-SignatureAlgorithms = ECDSA+SHA256:ed25519
+SignatureAlgorithms = ecdSA+SHA256:eD25519
```

Comment:
I would not make *any* of the changes here that vary the case - I would use the *official* names that we *want* users to work with - not aliases - but the preferred name,

Reply:
> I would not make _any_ of the changes here that vary the case - I would use the _official_ names that we _want_ users to work with - not aliases - but the preferred name,

The official names are used in the `.in` files, wrapped in `randcase()`. The case variants are machine-generated.